bigtable-test/bigtable-build-helper/src/it/verify-mirror-deps-ok/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright 2021 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud.bigtable.hbase.enforcer.it</groupId>
+  <artifactId>verify-mirror-deps-ok</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <description>Test project that ensures that build works when deps align</description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <hbase.version>1.4.3</hbase.version>
+  </properties>
+
+  <build>
```

Comment:
does this need skip deploy logic?

Reply:
no, these are test subjects that arent part of the regular maven build. The are used in the maven subprocess created by the invoker plugin